versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md

```diff
@@ -224,6 +224,15 @@ Sequelize also lets you pass credentials in URL format:
 const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
 ```
 
+You can pass tns alias, [connection descriptor](https://oracle.github.io/node-oracledb/doc/api.html#-1612-embedded-connect-descriptor-strings) and EZconnect to Sequelize constructor using `dialectoptions.connectString`:
```

Comment:
Use the same names as the node-oracledb doc, and correct the camel case of the attribute.
Change to "You can pass an Easy Connect String, a Net Service Name, or a Connect Descriptor to the Sequelize constructor using `dialectOptions.connectString`:"

If you want to give links, then give links for all of the three.  And use the 'official' links (see my other comment).

Reply:
I have updated the sentence accordingly. I have removed the link from here since the link for connectString is provided below.